packages/hoppscotch-common/src/components/app/Banner.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div
+    class="flex items-center px-4 py-2 text-tiny"
+    :class="bannerColor"
+    :role="bannerRole"
```

Comment:
```suggestion
    :role="bannerRole"
    class="flex items-center px-4 py-2 text-tiny"
    :class="bannerColor"
```

We can keep `class` towards the end of the list; that way there's a clear distinction.